proto/iot.proto

```diff
@@ -144,6 +165,32 @@ message ThermostatScheduleBlock {
     string time = 4;
 }
 
+message ThermostatADREvent {
```

Comment:
Can we make this a generic ADR event message? I don't think it needs to be specific to the thermostat

Reply:
That makes more sense, since it's not specific to thermostats. Changed the name to just DREvent! (I'm assuming Demand - Response, would it better to name it ADR?)

Reply:
Can you do DemandResponseEvent instead? No sense in abbreviating where we don't have to :)

Reply:
No problem, just changed! I also changed the enums at the top of the file to "DemandResponse" instead of DR.